0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -255,6 +256,7 @@ Messaging is used by connectors for [quoting](../0008-interledger-quoting-protoc
 
 Throws `InvalidFieldsError` if required fields are missing from the message or malformed.
 Throws `NotAcceptedError` if the message is rejected by the ledger.
+Throws `NoSubscriptionsError` if the message cannot be delivered because there is no corresponding websocket.
```

Comment:
Could be phrased more clearly, e.g.: Throws `NoSubscriptionsError` if the message cannot be delivered because there is nobody listening to messages addresses to the given account.